Array Manipulation/Program.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+
+namespace Array_Manipulation
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] arr = {55, 2, 1, 4, 0};
+
+            PrintArray(arr);
+
+            int[] arr6 = SelectionSort(arr);
+            PrintArray(arr6);
+
+            string textInput = PromptConsoleFor("a Number: ");
+            _ = int.TryParse(textInput, out int input);
+
+            int[] arr1 = InsertAtEnd(arr, input);
+            PrintArray(arr1);
+
+            int[] arr2 = InsertAtStart(arr, input);
+            PrintArray(arr2);
+
+            int[] arr3 = RemoveFromEnd(arr);
+            PrintArray(arr3);
+
+            int[] arr4 = RemoveFromStart(arr);
+            PrintArray(arr4);
+
+            
+            textInput = PromptConsoleFor("the place of the number you want to delete from the array: ");
+            _ = int.TryParse(textInput, out input);
+
+            int[] arr5 = RemoveAtIndex(arr, input);
+            PrintArray(arr5);
+        }
+
+        static string PromptConsoleFor(string text)
+        {
+            Console.Write("Enter {0}", text);
+            text = Console.ReadLine();
+
+            return text;
+        }
+
+        static void PrintArray(int[] arr)
+        {
+            Console.Write("Array: ");
+            foreach (var variable in arr)
+            {
+                Console.Write($"{variable} ");
+            }
+            Console.WriteLine();
+        }
+
+        static int[] SelectionSort(int[] arr)
+        {
+            int[] arrFinal = arr;
+            for (int i = 0; i < arr.Length - 1; i++)
+            {
+                int iMin = i;
+
+                for (int j = i + 1; j < arr.Length; j++)
+                {
+                    if (arrFinal[iMin] > arrFinal[j])
+                    {
+                        iMin = j;
+                    }
+                }
+
+                int temp = arrFinal[i];
+                arrFinal[i] = arrFinal[iMin];
+                arrFinal[iMin] = temp;
+            }
+            Console.Write("Array after Sorting: ");
+
+            return arrFinal;
+        }
+
+        static int[] InsertAtEnd(int[] arr, int value)
```

Comment:
Reuse InsertAt

Reply:
Sorry for late replies, the homework question was to add elements to specified location, do you need me to make a one functions that handles all the inserts and other function to handle all the deletion instead of making function for different parts in homework question?